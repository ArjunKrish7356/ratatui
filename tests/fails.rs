use ratatui_macros::constraints;

fn test() {
  // constraints![== 3, == 4; 10];
}
