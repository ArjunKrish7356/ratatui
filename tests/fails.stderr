warning: unused import: `ratatui_macros::constraints`
 --> tests/fails.rs:1:5
  |
1 | use ratatui_macros::constraints;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/fails.rs:5:2
  |
5 | }
  |  ^ consider adding a `main` function to `$DIR/tests/fails.rs`
